neotron-bmc-pico/src/main.rs

```diff
@@ -326,7 +336,7 @@ mod app {
 			_pin_uart_cts,
 			_pin_uart_rts,
 			led_power,
-			_buzzer_pwm,
+			speaker_pwm_tim: dp.TIM14,
```

Comment:
I think TIM14 is exclusively used by the speaker driver. If so, can we move this TIM14 object into some `speaker::Speaker` object to stop any other parts of the code fiddling with it?

Reply:
Good idea!